english/yoga/mind/yoga_mind.md

```diff
@@ -7,3 +7,9 @@ Want to know about Yoga lifestyle? Click [here](../lifestyle/yoga_lifestyle.md)
 Most people have discovered that exercising can lift their mood.
 
 Meditation is simply training your mind to focus and concentrate
+
+However, you are now calm and just as you begin to meditate you notice you are suddenly in a strange room with marshmallows for walls.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.